src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -513,6 +515,73 @@ public static class SystemNamespaceType
             return fileUri;
         }
 
+        private static TypeSymbol Base64ToStringTypeBuilder(IBinder binder, IFileResolver fileResolver, IDiagnosticWriter diagnostics, ImmutableArray<FunctionArgumentSyntax> arguments, ImmutableArray<TypeSymbol> argumentTypes)
```

Comment:
The type builders are created to provide different `armFunctionName` to `PerformArmConversionOfStringLiterals` only, but their signatures are identical. Maybe it's worth extracting a higher order function to parameterize `armFunctionName`?